src/components/FilmCard.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+      <b-card
+          :aria-hidden="show ? 'true' : null"
+          :title=title
+          :sub-title="originalTitle + ', ' + releaseDate.slice(0, 4)"
```

Comment:
I recommend you use back quotes with interpolation instead of concatenation. 
I didn't select all of cases when i faced with this syntax but if you decide to replace this style you can find the rest matches.

Reply:
v-bind syntax requires only this way of writting code like that